src/invidious/channels/videos.cr

```diff
@@ -101,7 +26,7 @@ module Invidious::Channel::Tabs
   end
 
   def get_videos(author : String, ucid : String, *, continuation : String? = nil, sort_by = "newest")
-    continuation ||= make_initial_content_ctoken(ucid, "videos", sort_by)
+    continuation ||= make_videos_ctoken(ucid, sort_by)
```

Comment:
Minor nitpick. Can you rename all of the `make_*_ctoken` methods here to `make_initial_*_ctoken`

Reply:
Sure thing!